lib/project_types/script/layers/domain/script_project.rb

```diff
@@ -17,6 +17,7 @@ class ScriptProject
         property! :language, accepts: String
 
         property :script_config, accepts: ScriptConfig
+        property :app_bridge, accepts: AppBridge
```

Comment:
Since during the project creation, we default `app_bridge` to `/`, is there a reason this property isn't required? Is it for backwards compatibility?

Reply:
Yeah it's for backwards compatibility. I don't want existing projects to break right away.